.github/workflows/reusable-build-docsite.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build the Ansible docsite
+
+on:
+  workflow_call:
+
+env:
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
+
+jobs:
+  build-docsite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install requirements
+        run: python -m pip install -r requirements.in -c requirements.txt
+      - name: Generate assets for the docs landing page
+        run: python -I -Werror build.py
+      - name: Copy sitemaps, redirects, and robots.txt files
+        run: cp -v *-sitemap.xml .htaccess robots.txt output/
```

Comment:
I'd expect `build.py` to include this piece of logic — this way, local builds are closer to the CI ones.

This is why people tend to use tox/nox — they are not just command runners — they wrap the env creation making everything as close as possible across dev environments of the contributors, CI, and prod.

Reply:
That's up to you really. I personally prefer small atomic changes. If you can do that, it'd be nice.

Reply:
This step was a hold over from the old approach where we needed to put web server stuff like the redirects on the same branch as the generated html. We can leave the sitemaps and things in this repo for visibility and can just do the cp step as part of the jenkins job.

I've removed this.

Reply:
Sounds like it could still belong in the build script, but under a CLI option. Though, that would probably require integrating a proper CLI parsing lib/framework.